### DISCUSSION TITLE
How do i disable discord/webcord wallbash intergration

### DISCUSSION THREAD

**Discussion by Im-shamo** [prasanthrangan/hyprdots#1545] [Q&A]
Wallpaper with light colors are changing discord to a white background which makes discord really unreadable and usable.

**Comment by kRHYME7** [comment]
remove this file  ` ~/.config/hyde/wallbash/Wall-Ways/discord.dcol `  or if you want to reuse this in the future, please change the ` .dcol` to any suffix other than  ` .dcol`
